--- istanbul-0.2.2/common/gst-feature.m4~	2008-10-20 20:02:31.978503574 +0200
+++ istanbul-0.2.2/common/gst-feature.m4	2008-10-20 20:02:59.602459247 +0200
@@ -46,13 +46,13 @@
 dnl GST_PLUGINS_NO will contain those that won't be built
 
 AC_DEFUN([GST_CHECK_FEATURE],
-echo
+[echo
 AC_MSG_NOTICE(*** checking feature: [$2] ***)
 if test "x[$3]" != "x"
 then
   AC_MSG_NOTICE(*** for plug-ins: [$3] ***)
 fi
-[dnl
+dnl
 builtin(define, [gst_endisable], ifelse($5, [disabled], [enable], [disable]))dnl
 dnl if it is set to NO, then don't even consider it for building
 NOUSE=
