--- istanbul-0.2.1/istanbul/main/screencast.py~	2006-07-28 17:52:17.000000000 +0200
+++ istanbul-0.2.1/istanbul/main/screencast.py	2006-11-09 00:43:26.706471000 +0100
@@ -93,7 +94,7 @@
             acappipeline = '%s ! audioconvert ! vorbisenc' % asource
 
         vencode_pipeline = 'video/x-raw-yuv,width=%d,height=%d,framerate=%s '\
-            '! theoraenc' % (width, height, framerate)
+            '! theoraenc quality=32' % (width, height, framerate)
 
         self.temp_file = tempfile.mkstemp()
 
